network-api/networkapi/wagtailpages/rss.py

```diff
@@ -14,8 +14,8 @@ class RSSFeed(Feed):
     """
 
     title = "Mozilla Foundation Blog"
-    link = "https://foundation.mozilla.org/blog/"
-    feed_url = "https://foundation.mozilla.org/blog/rss/"
+    link = "/blog/"
+    feed_url = "/blog/rss/"
```

Comment:
Hey Daniel, I think it would be better if we do this by fetching the actual page, something like:

```python
def get_link_url():
    blog_index_page = BlogIndexPage.objects.get(locale=Locale.get_default())
    return blog_index_page.url

...

link = get_blog_url()
feed_url = f"{get_blog_url()}/rss/"
```
This way, if `blog` ever changed then this would be up-to-date as well

Reply:
Thanks for the suggestion Jhonatan! 

I have implemented a get_blog_index_page() method in my latest commit and this PR is ready for re-review whenever you have time 👍 thanks again!